src/main/java/com/cb/services/DBService/DBServiceImpl/UserDBServiceImpl.java

```diff
@@ -55,11 +55,17 @@ public int insertUser(UserDAL u) {
         return template.query("SELECT * FROM users where email = '" + email + "'", new RowMapper<UserDAL>() {
             @Override
             public UserDAL mapRow(ResultSet rs, int rowNum) throws SQLException {
-                UserDAL u = new UserDAL();
-                u.setEmail(rs.getString(4));
-                u.setUserName(rs.getString(2));
-                return u;
+                try{
```

Comment:
I think it would be better if you  handle this exception in upper level service and put some message in DTO.